### PR TITLE
Conclusions

### LINKED ISSUE #27: Report - Conclusion Section
Use outline as a starting point, feel free to modify it as you see fit.

Others (mandatory):
  - [ ] Discussion on possible ethical issues with the full pipeline or your specific part of the pipeline

### PR DESCRIPTION
**Pull request by tristaaan**
Closes #27 

I have a lot of ideas on the ethical concerns part, but they're not all fully formed. 

WIP

### CODE REVIEW

**Comment by ankurshukla03** [comment]
```Moral issues can differ among person to person which can lead to be more controversial and chaotic but we need to develop the solution which should take care about ethical competence.```

Can this line go somewhere in the conclusion, i am trying to add it but couldn't go with the flow of the conclusion written.
Any suggestions?

**Comment by tristaaan** [comment]
Someone stop me.

**Comment by tristaaan** [comment]
Asking for review not because it's done, but because I could write another 1000 words and need some editors

File: report/sections/implementation.tex [hunk 1]
```diff
@@ -13,15 +13,15 @@ \section{Implementation} \label{impl}
 
 \subsection{kNN}
 
-A kNN classifier simply classifies samples by a majority vote of its neighbors, with a sample being assigned to the class most common among its \texttt{k} nearest neighbors \cite{data-mining-intro}. The hyper-parameter \texttt{n\_neighbors} determines the number of neighbors to use to choose the label of a sample, a total of 15 neighbors attained the best results.
+A kNN classifier simply classifies samples by a majority vote of its neighbors, with a sample being assigned to the class most common among its $k$ nearest neighbors \cite{data-mining-intro}. The hyper-parameter $n_neighbors$ determines the number of neighbors to use to choose the label of a sample, a total of 15 neighbors attained the best results.
```
Comment (diegocasmo) [inline report/sections/implementation.tex]: ``$n\_neighbors$``, though I still prefer ``\texttt{n\_neighbors}`` as this is not a math formula

Comment (tristaaan) [reply report/sections/implementation.tex]: but they're references to numeric variables in an algorithm

Comment (diegocasmo) [reply report/sections/implementation.tex]: [Code listing](https://cs.sharelatex.com/learn/Code_listing), also [When should one use \verb and when \texttt](https://tex.stackexchange.com/a/2791)

**Review by diegocasmo — APPROVED** [review]

**Review by ankurshukla03 — APPROVED** [review]

File: report/sections/implementation.tex (hunk 1 above)
Comment (tristaaan) [reply report/sections/implementation.tex]: reverting....

**Comment by tristaaan** [comment]
I want a Pulitzer nomination for that last section. This is ready to merge.

**Comment by ankurshukla03** [comment]
Not a nomination. Here is your [prize](https://www.google.se/search?biw=1535&bih=774&tbm=isch&sa=1&ei=mnMQW-jYDou7sQGYkILYCA&q=pulitzer+prize&oq=Pulitze&gs_l=img.3.1.35i39k1j0i67k1j0j0i67k1j0l2j0i67k1l2j0l2.38364.39208.0.41470.12.6.0.0.0.0.194.577.4j2.6.0....0...1c.1.64.img..10.2.303...0i24k1.0.F7xjXwwI1E8#imgrc=Jw3RqcJ-5avQcM:).

**Comment by tristaaan** [comment]
If only I knew how to print stuff here T_T .
ok, 'night.

-Tristan Wright

**Pull request merged**